web-app/src/app/screens/SignUp.tsx

```diff
@@ -372,30 +377,34 @@ export default function SignUp(): React.ReactElement {
           </p>
         </Box>
 
-        <Button
-          variant='outlined'
-          color='primary'
-          sx={{ mb: 2 }}
-          startIcon={<GoogleIcon />}
-          className='sso-button'
-          onClick={() => {
-            signInWithProvider(OauthProvider.Google);
-          }}
-        >
-          Sign Up With Google
-        </Button>
-        <Button
-          variant='outlined'
-          color='primary'
-          sx={{ mb: 2 }}
-          startIcon={<GitHubIcon />}
-          className='sso-button'
-          onClick={() => {
-            signInWithProvider(OauthProvider.Github);
-          }}
-        >
-          Sign Up With GitHub
-        </Button>
+        {enableGoogleSSO && (
```

Comment:
Let's revert this change before merging.

Reply:
✅